app/controllers/merchants_controller.rb

```diff
@@ -4,4 +4,7 @@ def index
 
   def show
   end
+
+  def items_index
```

Comment:
See comment below about routes -- I think it makes more sense to make our page in `merchants/items#index`

Reply:
Then we'd have to move the file `app/views/merchants/items_index.html.erb` to `app/views/merchants/items/index.html.erb`.